components/ui/loading-page.tsx

```diff
@@ -0,0 +1,12 @@
+'use client'
```

Comment:
Why not just throw this back up in loading.tsx (is there a need for it to be a seperate componenet?)